appveyor-nightly.yml

```diff
@@ -1,18 +1,18 @@
 configuration: Debug
 image: Visual Studio 2019
 
-environment:
-  app_version: "9.1"
-  APPVEYOR_BLOCK_DOTNETCORE_TESTS_AUTORUN: true
+version: '{build}'
```

Comment:
where is `version `used?

Reply:
This is a parameter that is used by appveyor